docs/cli.md

```diff
@@ -22,6 +22,13 @@ You can also find information on flags with `promscale_<version> -help`
 | web-enable-admin-api | boolean | false | Allow operations via API that are for advanced users. Currently, these operations are limited to deletion of series. |
 | web-listen-address | string | `:9201` | Address to listen on for web endpoints. |
 | web-telemetry-path | string | `/metrics` | Web endpoint for exposing Promscale's Prometheus metrics. |
+| tls-cert-file | string | "" | TLS Certificate file for web server, leave blank to disable TLS.
+| tls-key-file | string | "" | TLS Key file for web server, leave blank to disable TLS.
+| basic-auth-username | string | "" | HTTP Basic Authentication Username used for web endpoint authentication. Disabled by default.
+| basic-auth-password | string | "" | HTTP Basic Authentication Password used for web endpoint authentication. Password or password-file option should be set together with username.
+| basic-auth-password-file | string | "" | HTTP Basic Authentication Password file containing the actuall password used for web endpoint authentication. Password or password-file option should be set together with username.
```

Comment:
```suggestion
| auth-password-file | string | "" | Address for auth password file containing the actual password used for web endpoint authentication. This flag should be set together with auth-username. It is mutually exclusive with auth-password and bearer-token methods. |
```

Reply:
Should start "Path to auth password file."